gno.land/pkg/integration/testing_node.go

```diff
@@ -147,6 +150,16 @@ func LoadDefaultGenesisBalanceFile(t TestingTS, gnoroot string) []gnoland.Balanc
 	return genesisBalances
 }
 
+// LoadDefaultGenesisParamFile loads the default genesis balance file for testing.
+func LoadDefaultGenesisParamFile(t TestingTS, gnoroot string) []gnoland.Param {
+	paramFile := filepath.Join(gnoroot, "gno.land", "genesis", "genesis_params.toml")
```

Comment:
No need to change it in this PR, but for the record I really think we should remove this context-based file loading from any test we have, integration or otherwise :/ 

It's gonna become difficult to maintain when files are moved around, especially if you consider the `genesis` folder doesn't contain anything required for gno chains, just generated data